task_sdk/src/airflow/sdk/definitions/baseoperator.py

```diff
@@ -1418,6 +1418,11 @@ def inherits_from_empty_operator(self):
         # of its subclasses (which don't inherit from anything but BaseOperator).
         return getattr(self, "_is_empty", False)
 
+    @property
+    def inherits_from_skip_mixin(self):
+        """Used to determine if an Operator is inherited from SkipMixin."""
+        return getattr(self, "_is_skip_mixin", False)
```

Comment:
If you put this on AbstractOperator then you don't need to change MappedOperator as much (just define the alias/attribute)

Reply:
Done, had to add equivalent definitions for the decorators (otherwise DAG proc. gets mad)